// statistics_service.proto

syntax = "proto3";

import "google/protobuf/timestamp.proto";

service StatisticsService {
  rpc AddLike(ActionRequest) returns(ActionResponse);
  rpc AddView(ActionRequest) returns(ActionResponse);
}

message ActionRequest {
  int32 post_id = 1;
  int32 user_id = 2;
}

message ActionResponse {
  int32 post_id = 1;
  int32 user_id = 2;
  google.protobuf.Timestamp updated_at = 3;
}